cmd/syncthing/gui.go

```diff
@@ -1017,6 +1020,30 @@ func (s *apiService) getEvents(w http.ResponseWriter, r *http.Request) {
 	sendJSON(w, evs)
 }
 
+func (s *apiService) getDiskEvents(w http.ResponseWriter, r *http.Request) {
```

Comment:
Given that this is identical to the other events handler, apart from the "gotEventRequest" comment below, I think it should be refactored into a common function/method that just takes the relevant event subscription as a parameter.